pkg/envinfo/envinfo.go

```diff
@@ -26,6 +26,14 @@ type ComponentSettings struct {
 	URL       *[]EnvInfoURL `yaml:"Url,omitempty"`
 }
 
+// URLType is an enum to indicate the type of the URL i.e ingress/route
+type URLKind string
+
```

Comment:
nit: URLType is more better.

Reply:
@kadel suggested URLKind so I used it. Also I have fixed the comment.